### ISSUE TITLE
No configuration available for Fibaro FGRM222

### ISSUE THREAD

**Issue by Oohcalme** [OpenZWave/Zwave2Mqtt#429]
**1.6**

Build/Run method
Zwave2MQTT Addon on Home Assistant

**Describe the bug**
No configuration option available for the FIBARO FGRM222 whereas the device exists on openzwave database (checked id and type).
Other devices are ok, except the ZWave Stick from Aeotec with no configuration 

**Additional context**
the only difference I can see betwen the FIBARO FGRM222 and othe devices is that the code display in decimal is only 3 digits long (same for the zwave stick), maybe a problem with conversion ? (sorry it's just a guess, i'm not a programmer...)
Other information, it worked before (I can't say on which version unfortunately) and in the device list the type is not display for these devices (others are all ok)

Thank you for your help

**Comment by robertsLando** [comment]
I don't know which Z2M/OZW specific version the addon is using

**Comment by Oohcalme** [comment]
it uses Zwave2Mqtt v3.0.2 in its last release.

**Comment by robertsLando** [comment]
Did you enabled the zwave setting to auto update devices db? 

Did you try to send a refresh node info against that node?

**Comment by Oohcalme** [comment]
Yes I tried. Auto update db is on and the refresh node info does nothing.
I have 10 FGRM222 and I just saw that the two last one I added recently have the configuration.
I guess I will have to exclude/include the others again (that's a nightmare as I can add them one by one only)

Except if there's something to solve it without include them again, let's close the point !
thank you  Roberts for the answers (and everything else)
